docs/sections/instructionsFile.md

```diff
@@ -1,58 +1,60 @@
 [Back to Table of Contents](../documentation.md)
 
-# Instructions File YAML Schema
+# Instructions file YAML schema
 
-What is an instructions file?  A YAML document that contains an array of paths to YAML documents to perform operations on.  Each instructions file starts with the optional key `commonOverlays`, which is an array, along with a required key `yamlFiles` which is also an array.  The instructions file is the instructions for manipulating a set of YAML files and documents.
+What is an instructions file?  
 
-## Top-level commonOverlays Keys
+The instructions file is the instructions for manipulating a set of YAML files and documents. It is a YAML document that contains an array of paths to other YAML documents where you can perform operations.  Each instructions file starts with the optional key `commonOverlays`, which is an array, along with a required key `yamlFiles`, which is also an array.  
 
-The `commonOverlays` key is completely optional, and is a means to providing overlays that should be applied to every item in the `yamlFiles` array or in other words, `yamlFiles[*].path` or `yamlFiles[*].documents.path`entries defined in the instructions. Each array item in the `commonOverlays` key is treated as a dictionary/map with the following top-level keys:
+## Top-level commonOverlays keys
 
-| key | required | description | default | type |
+The `commonOverlays` key is optional. It provides overlays to apply to every item in the `yamlFiles` array, also referred to as `yamlFiles[*].path` or `yamlFiles[*].documents.path` as defined in the instructions. Each array item in the `commonOverlays` key is treated as a dictionary/map with the following top-level keys:
+
+| **Key** | **Required** | **Description** | **Default** | **Type** |
 | --- | --- | --- | --- | --- |
-| name | no | An optional description of the change you are performing, and used for either on-screen output or self-documentation. | None | string |
```

Comment:
Just need the keys in the table flipped back to all lowercase and the rest is all good.